requirements.txt

```diff
@@ -9,4 +9,7 @@ uvloop>=0.14.0
 grpcio>=1.33.2
 grpcio-tools>=1.33.2
 protobuf>=3.12.2
-configargparse>=1.2.3
\ No newline at end of file
+configargparse>=1.2.3
+trio==0.16.0
```

Comment:
nit: Replace `==` to `>=` to allow updating `trio` and modules depending on it?

Reply:
You have to install exactly this version to make p2pclient work with python 3.8. Otherwise import errors arise.